src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProject.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OpenApi.Models;
+
+    public class GetProject
+    {
+        private readonly IProjectsService projectsService;
+
+        public GetProject(IProjectsService projectsService)
+        {
+            this.projectsService = projectsService;
+        }
+
+        [FunctionName("GetProject")]
```

Comment:
As it stands right now, this is not a RESTful API. Let's adhere to resources naming good practices, ok? Your endpoint should be something like `GET api/projects/{id}` :) 
Read more here: https://restfulapi.net/resource-naming/

Big refactor incoming...

Reply:
the endpoint to getProject is something like that:
Get api/project/{id}

![image](https://user-images.githubusercontent.com/48032651/126530103-44135774-c009-4d65-9631-ce1068abebc4.png)

Reply:
```suggestion
       [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route ="projects/{id:guid}")] HttpRequest req)
```

Reply:
@Raul-Gamarra-jalafund here man